ideascube/conf/base.py

```diff
@@ -283,3 +283,6 @@
 
 SESSION_COOKIE_AGE = 60 * 60  # Members must be logged out after one hour
 BACKUP_FORMAT = 'zip'  # One of 'zip', 'tar', 'bztar', 'gztar'
+
+CATALOG_CACHE_BASE_DIR = '/tmp/ideascube-catalog'
```

Comment:
> Not a solution for the developer workstation, though.

Can be managed by `dev.py` dedicated settings. Also having a default in `STORAGE_ROOT/package_cache` for example.